templates/package/plopfile.mjs

```diff
@@ -14,6 +14,10 @@ const exec = util.promisify(childProcess.exec);
 
 const workspaceRoot = getWorkspaceRoot(process.env.PWD);
 const packagesPath = path.resolve(workspaceRoot, "packages");
+const packageInfo = getPackageInfos(packagesPath);
+
+const tsConfigVersion = packageInfo["@pcd/tsconfig"].version;
+const eslintConfigVersion = packageInfo["@pcd/eslint-config-custom"].version;
```

Comment:
it could also be prudent to identify and fix all packages whose names are prefixed with `@pcd/`, or warn in the case that auto-fixing is annoying to implement.

another thing that could work is to use the yarn workspaces 'cross-references' feature - https://yarnpkg.com/features/workspaces#cross-references. this would mean that using the right version is simply not necessary at development time, and the versions get auto-filled at publish time (as far as I understand). However, i suspect this may not interact well with the `changeset` package that we're using to update package versions in lockstep.

anyways, take all this mostly as musing out-loud, what's here is good enough for now imo.

Reply:
Ok. The reason I only included these two packages is that they're the only ones in the template `package.json`. If you created a package from the template and added other dependencies to it, that package would be part of the workspace and would have its dependencies updated by the changeset mechanism.